plugin-management-cli/pom.xml

```diff
@@ -4,12 +4,90 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <artifactId>plugin-manager</artifactId>
-        <groupId>jenkins</groupId>
+        <groupId>io.jenkins</groupId>
         <version>1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
```

Comment:
Move above `parent`

Reply:
Do you mean move the modelVersion? Fixed in later commit.